app/controllers/finance/participants_controller.rb

```diff
@@ -14,15 +14,24 @@ def index
       end
     end
 
-    def show; end
+    def show
```

Comment:
we have a [ticket](https://dfedigital.atlassian.net.mcas.ms/jira/software/projects/CPDLP/boards/87?selectedIssue=CPDLP-3590) to address finance changes, can we please exclude those from this pr?